src/test/java/net/uaznia/lukanus/hudson/plugins/gitparameter/GitParameterDefinitionTest.java

```diff
@@ -398,8 +398,13 @@ public void testDoFillValueItems_listRevisions() throws Exception {
         assertNotNull(build);
         ItemsErrorModel items = def.getDescriptor().doFillValueItems(project, def.getName());
         assertTrue(isListBoxItem(items, "00a8385cba1e4e32cf823775e2b3dbe5eb27931d"));
-        assertTrue(isListBoxItemName(
-                items, "00a8385c 2011-10-30 17:11 Łukasz Miłkowski <lukanus@uaznia.net> initial readme"));
+        if (!Functions.isWindows() || System.getenv("CI") == null) {
```

Comment:
https://github.com/jenkins-infra/helpdesk/issues/4568#issuecomment-2710736827 Rather than checking either the OS or the CI/local environment, I would think this test should either be fixed to pass in whatever encoding it happens to be run in, or to skip this assertion (or use `Assume` to skip the test case) in case that is impossible for some reason. I am not familiar enough with the plugin to have an opinion which it should be.

Reply:
I've adjusted the test to remove the character set dependency.  I don't think that was ever the intent of the test.

* https://github.com/jenkinsci/git-parameter-plugin/pull/150